plugins/obs-ffmpeg/data/locale/en-US.ini

```diff
@@ -1,6 +1,11 @@
 FFmpegOutput="FFmpeg Output"
 FFmpegAAC="FFmpeg Default AAC Encoder"
 FFmpegOpus="FFmpeg Opus Encoder"
+FFmpegALAC="ALAC (24-bit)"
+FFmpegFLAC="FLAC (16-bit)"
+FFmpegPCM16Bit="PCM (16-bit)"
+FFmpegPCM24Bit="PCM (24-bit)"
+FFmpegPCM32BitFloat="PCM (32-bit float)"
```

Comment:
Even if there is low chance to have non-FFmpeg alternative of those codec as first-party.

Like AAC and Opus, all of them should have "FFmpeg" inside of their names